mlrun/runtimes/function.py

```diff
@@ -484,7 +490,7 @@ def with_node_selection(
         node_selector: typing.Optional[typing.Dict[str, str]] = None,
         affinity: typing.Optional[client.V1Affinity] = None,
     ):
-        raise NotImplementedError("Node selection is not supported for nuclio runtime")
+        super().with_node_selection(node_name, node_selector, affinity)
```

Comment:
If you're not doing anything except calling the base implementation you can just remove it